src/components/pages/ChildDashboard/RenderChildDashboard.js

```diff
@@ -5,8 +5,8 @@ import { useHistory } from 'react-router-dom';
 import { InstructionsModal } from '../../common';
 import { modalInstructions } from '../../../utils/helpers';
 import { HUD } from '../HeadsUpDisplay/index';
-
-import adventure_passport from '../../../assets/images/child_dashboard_images/adventure_passport.svg';
+import Explosion from '../../../assets/images/gamemodeimg/explosion.png';
+// import adventure_passport from '../../../assets/images/child_dashboard_images/adventure_passport.svg';
```

Comment:
This import should be deleted instead of commented out.

Reply:
done